frontend/packages/core/src/accordion.tsx

```diff
@@ -77,36 +78,92 @@ export const StyledAccordionSummary = styled(AccordionSummaryBase)({
   },
 });
 
-export interface AccordionProps extends Pick<MuiAccordionProps, "defaultExpanded"> {
+const StyledAccordionGroup = styled.div({
+  ".MuiAccordion-root": {
+    marginBottom: "16px",
+  },
+
+  ".MuiAccordion-root.Mui-expanded": {
+    marginBottom: "16px",
+  },
+
+  ".MuiAccordion-root:before": {
+    display: "none",
+  },
+});
+
+export interface AccordionProps extends Pick<MuiAccordionProps, "defaultExpanded" | "expanded"> {
   title?: string;
   collapsible?: boolean;
-  children: React.ReactNode[];
+  children: React.ReactNode | React.ReactNode[];
```

Comment:
`ReactNode` should accept an array of ReactNodes so I think you can do:
```suggestion
  children: React.ReactNode;
```